### PR TITLE
Blog post on affiliations (MT & CSG)

### PR DESCRIPTION
**Pull request by seasidesparrow**

### CODE REVIEW

File: blog/_posts/2021-04-15-affils-update.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+There are comparatively few publications by authors affiliated with these colleges: Salish Kootenai College (Montana), Northwest Indian College (Pacific NW), Din&eacute; College (Arizona) and Fond du Lac Tribal College (Minnesota) have the most publications. Papers written by TCU authors and coauthors in the ADS primarily focus on topics in Environmental Science and Science Education, which are not currently part of our core literature.  These institutions may become more frequently represented if our scope widens to include more related sciences (e.g. planetary and allied literature).  We also want to ensure that their contributions are noted in existing literature, and that their contributions will be more easily tracked if these institutions increase their participation in the ADS core fields of astronomy and physics.
```
Comment (kelockhart) [inline blog/_posts/2021-04-15-affils-update.md:131]: The colon in the first sentence makes me think the four colleges listed have few publications, but the end of the sentence makes it sound like those 4 have the most. Can you clarify? If it's that these are the most represented, changing the colon to a period or semi-colon is all that's needed (probably a period - there are a lot of semi-colons in this subsection already).

File: blog/_posts/2021-04-15-affils-update.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+If you find affiliation IDs that aren't correct, please let us know.  Likewise if there are affiliations that you think should be included or better disambiguated as parents and children, please get in touch.  We would especially welcome help with the assignment and disambiguation of French UMRs and Universities, the Russian and Chinese Academies of Science, and any institutions from the Global South especially if they conduct published research in the ADS core fields of Astronomy and Physics.
```
Comment (kelockhart) [inline blog/_posts/2021-04-15-affils-update.md:137]: I like this specific call for action/help! Add a link to the adshelp@ email somewhere in this paragraph.

**Review by kelockhart — CHANGES_REQUESTED** [review]
Looks really good! I have a few formatting comments, plus just a couple readability ones. Otherwise it's about good to go.

File: blog/_posts/2021-04-15-affils-update.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+**Tribal Colleges and Universities (TCUs)**
+
+We have also added affiliation IDs for a partial list of Tribal Colleges and Universities (TCUs), primarily those in the United States that are currently represented in the ADS Literature.  The list of identified TCUs can be accessed [here](https://github.com/adsabs/CanonicalAffiliations/blob/master/tribal_colleges_universities.tsv); note that this list includes only those institutions designated as tribally-controlled, and does not include those public or private non-Tribal institutions having a significant Native American population; some of those schools are present in the list as public universities, however, mainly those that have science and engineering programs.
```
[the remark this answers is not in the document]
Comment (seasidesparrow) [reply blog/_posts/2021-04-15-affils-update.md:129]: Modified slightly, and added a short list of non-tribal colleges with significant native american populations.

File: blog/_posts/2021-04-15-affils-update.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+* *aff_id*: This field can be used to search for affiliation identifications curated by the ADS, including synonyms with other institutional identifier systems like GRID and ROR.  It requires you to use the controlled dictionary of identifiers specified <here> (https://github.com/adsabs/CanonicalAffiliations/blob/master/parent_child.tsv).  It does not expand the search to include an institution's children.
+
+* *inst*: This field is a shortcut to enable you to search for institutions without requiring you to know their parent.  It also allows you to search the affiliation identifications curated by the ADS, including synonyms with other institutional identifier systems like GRID and ROR.  Unlike aff_id, it expands the search to include an institution's children.  It is convenient for unique institutions such as the "CfA".  However, for an institution that is not unique, such as "Inst Phy", you will want to include the parent to disambiguate.
```
[the remark this answers is not in the document]
Comment (seasidesparrow) [reply blog/_posts/2021-04-15-affils-update.md:57]: I expanded the discussion under "What are affiliations" to introduce the concept of parents and children.

**Pull request merged**